README.md

```diff
@@ -36,6 +36,20 @@ We will focus on building an open API first. Then developers can use the API to
 - [Ideas for Version 1 or Version 2 of the App](https://github.com/freeCodeCamp/chapter/issues/1)
 - [Introduce Yourself / Volunteer Your Skills](https://github.com/freeCodeCamp/chapter/issues/11)
 
+### Terminology
+
+To better communicate and more easily build an API and UI, we've decided on a collection of terminology to describe Chapter in a clear way.
+
+- A "collective" is one singular instance/deployment of Chapter
+  *For example, Women who Code at the domain chapter.womenwhocode.com*
```

Comment:
```suggestion
  *For example, Women who Code at the domain `chapter.womenwhocode.com`*
```

[URLs are backtick'd above](https://github.com/freeCodeCamp/chapter#welcome-to-chapter), best to do so here as well.